src/testdir/Make_all.mak

```diff
@@ -278,6 +278,7 @@ NEW_TESTS = \
 	test_spell \
 	test_spell_utf8 \
 	test_spellfile \
+	test_spellrare \
```

Comment:
You also need to add it below to NEW_TESTS_RES target

Reply:
Fixed